components/organisms/Feedback.js

```diff
@@ -0,0 +1,188 @@
+            <h2 className="text-h4 lg:text-h3 lg:text-sm font-display mt-2 mb-4 w-48 sm:w-auto">
+              {t("improveService")}
+            </h2>
+            <button
+              id="feedbackClose"
+              onClick={closeFeedback}
+              className="font-body text-white flex text-left sm:ml-4 my-2 sm:my-0"
+              aria-haspopup="true"
+              aria-expanded="false"
+              aria-controls="feedbackDropdown"
+              data-testid="closeButton"
+            >
+              <span
+                id="close"
+                className="text-h3 lg:text-h2 leading-4 lg:leading-10"
+              >
+                &times;
+              </span>
+              <span className="text-xs leading-4 lg:text-sm underline ml-2 lg:leading-10">
+                {t("close")}
+              </span>
+            </button>
+          </div>
+          <ul className="list-outside list-disc px-6 py-2">
+            <li className="text-xs lg:text-sm font-body mb-4">
+              <b>{t("reportAProblemNoReply", { lng: props.language })}</b>{" "}
+              {t("reportAProblemEnquiries", { lng: props.language })}{" "}
+              <a
+                className="underline text-xs lg:text-sm font-body hover:text-gray-300 text-white"
+                href="mailto:experience@servicecanada.gc.ca"
+              >
+                experience@servicecanada.gc.ca
+              </a>
+              .
+            </li>
+            <li className="text-xs lg:text-sm font-body mb-4">
+              <b>{t("confidential", { lng: props.language })}</b>
+              <ActionButton
+                id="link-privacyPage"
+                dataCy="link-privacyPage"
+                dataTestId="link-privacyPage"
+                href={t("privacyLink")}
+                text={t("reportAProblemPrivacyStatement")}
+                custom="text-xs lg:text-sm underline ml-2 hover:text-gray-300"
+              />
+            </li>
+          </ul>
+          <form
+            data-gc-analytics-formname="ESDC|EDSC:ServiceCanadaLabsFeedback-Form"
+            data-gc-analytics-collect='[{"value":"input,select","emptyField":"N/A"}]'
+            className="w-full"
+            action="#"
+            onSubmit={onSubmitHandler}
+          >
+            <label
+              htmlFor="feedbackTextArea"
+              className="text-xs lg:text-sm font-body font-bold"
+            >
+              {t("doBetter")}
+            </label>
+            <p className="text-xs lg:text-sm my-2">{t("doNotInclude")}</p>
+            <p className="text-xs lg:text-sm my-2">{t("maximum2000")}</p>
+            <textarea
+              id="feedbackTextArea"
+              name="feedbackTextArea"
+              maxLength="2000"
+              rows="5"
+              className={
+                "text-input font-body w-full min-h-40px shadow-sm text-form-input-gray border-2 py-6px px-12px rounded"
+              }
+              required
```

Comment:
"required" means that the browser validation takes over, which we tend to get rid of and handle using our error messages. We should remove this and use the error message / validation message pattern we already have for the signup page.